arbitrum-docs/node-running/quickstart-running-a-node.md

```diff
@@ -0,0 +1,39 @@
+---
+title: "Quickstart: Run different types of Arbitrum nodes"
```

Comment:
I would change the title to "Run an Arbitrum node" to be consistent with the section name "Run a node".
Menu entries are also inconsistent, some say "Arbitrum nodes" and other "Run a node". I would change the former to the latter, as I find "Run a node" more precise.